### ISSUE TITLE
Cacti 1.1.38 - Enable changing "selected graph template" in Graphs Management

### ISSUE THREAD

**Issue by sysres-dev** [Cacti/cacti#2560]
Our Cacti users complained about some behaviour changes in the interface in Cacti 1.1.38 because we used Cacti 0.8.8 for a long time, and they would like some behaviours closer to it :
- On the Graphs Management, the item "Selected Graph Template" is not changeable.
Our users used to replace a graph template by an other.
For example a Graph for 32 CPU was able to be replaced by a Graph for 40 CPU in Cacti 0.8.8 but not in Cacti 1.1.38 because the item is locked.

In Cacti 0.8.8 there was all the graph templates listed in the "Selected Graph Template" dropdown list.

It would be nice to have the same behaviour like in Cacti 0.8.8.

**Comment by cigamit** [comment]
Well, we changed that function to not allow you to change the template from a Data Source that does not have the exact number of Data Sources, and the exact matching Data Source names.

I know what you are talking about, but we can only do that once we have a function to re-organize the RRDfiles to include the missing internal data source names.

So, this is a feature request.  We used to allow it, but it would break Graphs.
